bears/general/SpaceConsistencyBear.py

```diff
@@ -38,7 +41,47 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
-        for line_number, line in enumerate(file, start=1):
+        def get_blanklines_nr_start():
+            line_nr_start = False
+            rev_enumerated_zip_obj = zip(reversed(range(1, len(file) + 1)),
+                                         reversed(file))
+            for line_number, line in rev_enumerated_zip_obj:
+                replacement = line
+                if replacement.strip() == '':
+                    line_nr_start = line_number
+                else:
+                    break
```

Comment:
both `if` and `else` can be written inline.

Reply:
I am not confirm we should do this or not?

Reply:
does it violate any bear?

Reply:
as far i remember when theres only one line in if statement it is good to inline it ;)